bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttService.java

```diff
@@ -387,6 +410,7 @@ public void unregisterMessageProducer(String brokerName, MqttMessageProducer com
      * @param eventPublisher EventPublisher
      */
     @Deprecated
+    @Reference(cardinality = ReferenceCardinality.MANDATORY, policy = ReferencePolicy.DYNAMIC)
```

Comment:
I assume we don't try to solve the "dynamic so volatile" topic here. Correct? It has been this way before and so we leave it for now...